python_modules/libraries/dagster-fivetran/dagster_fivetran/asset_defs.py

```diff
@@ -129,6 +143,10 @@ def build_fivetran_assets(
             table name to user-supplied metadata that should be associated with the asset for that table.
         group_name (Optional[str]): A string name used to organize multiple assets into groups. This
             group name will be applied to all assets produced by this multi_asset.
+        materialize_missing_tables (bool): If True, will create asset materializations for tables specified
```

Comment:
kinda confusing name, as this option does not actually change what gets materialized. instead, it just creates events. maybe something like "infer_missing_tables" or something like that? hard to think of a not totally-verbose name